dluhc-component-library/src/mockData/timetable.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "developmentPlanEvent": "Scoping of the New Local Plan",
```

Comment:
The actual format, keys and values of this file are not final and may be updated in the future